gbpservice/nfp/configurator/agents/agent_base.py

```diff
@@ -157,9 +164,10 @@ def process_batch(self, ev):
                 resource_data = request['resource_data']
                 agent_info = request['agent_info']
                 resource = agent_info['resource']
+                # agent_info contains the API context.
                 context = agent_info['context']
                 service_vendor = agent_info['service_vendor']
-                service_type = agent_info['service_type']
+                service_type = agent_info['resource_type']
```

Comment:
I thought about this. The issue is that in the notification data that is sent,
the info dict needs to contain 
{ 'service_type': resource_type}. Either way, this mapping will be there.